.travis.yml

```diff
@@ -114,6 +115,37 @@ jobs:
       region: eu-west-1
       on:
         branch: development
+  - stage: Platform Deploy Beta
+    if: NOT type IN (pull_request)
+    env:
+    - NODE_ENV=production
+    - PORT=12345
+    - STREAMR_API_URL=https://www.streamr.com/api/v1
+    - STREAMR_WS_URL=wss://www.streamr.com/api/v1/ws
+    - PLATFORM_ORIGIN_URL=https://www.streamr.com
```

Comment:
PLATFORM_ORIGIN_URL | Base path/address of the current environment

`PLATFORM_ORIGIN_URL` -> `https://beta.streamr.com`